ui/app/AppLayouts/Wallet/components/SetCurrencyModalContent.qml

```diff
@@ -63,13 +64,19 @@ Item {
 
                 MouseArea {
                     anchors.fill: parent
+                    cursorShape: Qt.PointingHandCursor
                     hoverEnabled: true
                     onEntered: {
                         wrapper.hovered = true
                     }
                     onExited: {
                         wrapper.hovered = false
                     }
+                    onClicked: {
+                        currencyRadioBtn.checked = !currencyRadioBtn.checked
+                        modalBody.currency = key
```

Comment:
This is a bit of an optimistic update here, considering that `walletModel.setDefaultCurrency` in the next line could potentially fail. But I've used this as a quick route to get `modalBody.currency` reflect the latest value, as this is needed to render the proper selected item.

The alternative would be to set up a `Connection` on `defaultCurrency` and update the prop from the outside every time it changes, so that the list is re-rendered.